src/components/LeftSidebar/NewGroupConversation/SetContacts/SetContacts.vue

```diff
@@ -127,11 +137,14 @@ export default {
 		},
 	},
 
-	async mounted() {
-		// Focus the input field of the current component.
-		this.focusInput()
-		// Perform a search with an empty string
-		await this.fetchSearchResults()
+	mounted() {
+		this.$nextTick(() => {
+			this.mountArrowNavigation(this.$refs.wrapper, this.$refs.setContacts)
+			// Focus the input field of the current component.
+			this.focusInput()
```

Comment:
Pressing Esc in the modal will close it.

I suggest to also add the Escape behavior from useArrowNavigation when the focus is on the search field or on an item from the participants list. Otherwise, the closing modal applies.